migrations/v8-MIGRATION.md

```diff
@@ -0,0 +1,48 @@
+# V8 API Migration Guide
+
+This latest release updates the SDK to use BillingClient 7. This version of BillingClient brings little change compared
+with BillingClient 5 which brought an entire new subscription model which resulted in large changes across the entire SDK. It did, however, increase the minSdk to 21 (Android 5.0) and removed the `ProrationMode` enum.
```

Comment:
I don't think there's much value in talking about BC5 at this stage. If, as a customer, I've been using v7 since release, what do I care how BC6 -> BC7 compares as a transition to BC4 -> BC5? 

I would instead simply acknowledge the 2 new things in BC7: pending transactions for prepaid plans and installments. 

And add a link to where people can find out more about them.

Reply:
Let's also add a code snippet that showcases how to use the 2 new things, or at least link to relevant docs

Reply:
to avoid duplicating work, I also opened a [PR](https://github.com/RevenueCat/docusaurus/pull/279) for the main doc page that includes details about installments and pending transactions. Code snippet would be super helpful!